src/DataFrames.jl

```diff
@@ -12,7 +12,8 @@ using Compat
 import Compat.String
 using Reexport
 @reexport using StatsBase
-@reexport using DataArrays
+@reexport using NullableArrays
```

Comment:
Are the `@reexport`s necessary? Assuming I didn't miss anything reading through this, you're only overloading one NullableArrays function and two CategoricalArrays functions. Could we just export the modified functions rather than reexporting the entire modules?